packages/block-library/src/gallery/gallery-image.js

```diff
@@ -134,12 +134,14 @@ class GalleryImage extends Component {
 							onClick={ onMoveBackward }
 							className="blocks-gallery-item__move-backward"
 							label={ __( 'Move Image Backward' ) }
+							disabled={ isFirstItem }
```

Comment:
I think "disabled" is not recommended across the UI to avoid focus loss when the button becomes disabled. Instead we just use `aria-disabled`  and an empty onClick event handler.

I wonder if this is something that we should bake in `Button` component?

Reply:
Fixed at bfcc397